kernel/src/process.rs

```diff
@@ -916,8 +929,10 @@ impl<'a> Process<'a> {
               \r\n  R4 : {:#010X}    R11: {:#010X}\
               \r\n  R5 : {:#010X}    R12: {:#010X}\
               \r\n  R9 : {:#010X} (Static Base Register)\
-              \r\n  PC : {:#010X} [{:#010X} in lst file]\
+              \r\n  SP : {:#010X} (Process Stack Pointer)\
               \r\n  LR : {:#010X} [{:#010X} in lst file]\
+              \r\n  PC : {:#010X} [{:#010X} in lst file]\
+              \r\n YPC : {:#010X} [{:#010X} in lst file]\
```

Comment:
Good question. I don't have a solid answer because I don't really know. Opened #244 so we don't lose track.